tts-python-evaluation-QA.ipynb

```diff
@@ -0,0 +1,439 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fe74c6db",
+   "metadata": {},
+   "source": [
+    "# TTS Evaluation QA"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a833017",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will use  Automatic speech recognition(ASR) to generate transcripts from TTS synthesized data and compare the generated transcripts against the groundtruth using edit distance and Word error rate (WER).\n",
```

Comment:
Let's stick to character error rate instead of edit distance.
So instead of `against the groundtruth using edit distance and Word error rate (WER).` do `against the groundtruth using character error rate (CER) and word error rate (WER). Character error rate is also referred to as edit distance`

And in the rest of the notebook, replace edit distance with character error rate

Reply:
Character error rate is (edit distance)/(num words). I have changed edit distance to CER, but I have kept it as edit distance where required.

Reply:
Thanks for that correction